src/zelos/util.py

```diff
@@ -85,6 +85,10 @@ def dumpstruct(struct_obj, indent_level=0):
         except Exception:
             pass
         print(f"{indent}{field[0]}: {val}")
+        try:
+            dumpstruct(val, indent_level=indent_level + 1)
+        except Exception:
+            pass
```

Comment:
We should print out (either an error in the output string or an error in logging)